app/renderer/components/preferences/payment/captcha.js

```diff
@@ -0,0 +1,203 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const {StyleSheet, css} = require('aphrodite/no-important')
+
+// Components
+const ImmutableComponent = require('../../immutableComponent')
+
+// Actions
+const appActions = require('../../../../../js/actions/appActions')
+
+// Constants
+const promotionStatuses = require('../../../../common/constants/promotionStatuses')
+
+// Styles
+const cx = require('../../../../../js/lib/classSet')
+const closeButton = require('../../../../../img/toolbar/stoploading_btn.svg')
+const dragIcon = require('../../../../extensions/brave/img/ledger/BAT_captcha_dragicon.png')
+const arrowIcon = require('../../../../extensions/brave/img/ledger/BAT_captcha_BG_arrow.png')
+
+// TODO: report when funds are too low
+class Captcha extends ImmutableComponent {
+  constructor (props) {
+    super(props)
+    this.onCaptchaDrop = this.onCaptchaDrop.bind(this)
+    this.onCaptchaDrag = this.onCaptchaDrag.bind(this)
+    this.getText = this.getText.bind(this)
+    this.captchaBox = null
+    this.dndStartPosition = {
+      x: 0,
+      y: 0,
+      width: 0,
+      height: 0
+    }
+  }
+
+  onCaptchaDrop (event) {
+    event.preventDefault()
+    const target = this.captchaBox.getBoundingClientRect()
+
+    const x = event.clientX - target.left - this.dndStartPosition.x + (this.dndStartPosition.height / 2)
+    const y = event.clientY - target.top - this.dndStartPosition.y + (this.dndStartPosition.width / 2)
+
+    appActions.onPromotionClaim(x, y)
+  }
+
+  onCaptchaDrag (event) {
+    const target = event.target.getBoundingClientRect()
+    this.dndStartPosition = {
+      x: event.clientX - target.left,
+      y: event.clientY - target.top,
+      width: target.width,
+      height: target.height
+    }
+  }
+
+  preventDefault (event) {
+    event.preventDefault()
+  }
+
+  closeCaptcha () {
+    appActions.onCaptchaClose()
+  }
+
+  getText () {
+    if (this.props.promo.get('promotionStatus') === promotionStatuses.CAPTCHA_ERROR) {
+      return {
+        title: 'promotionCaptchaErrorTitle',
+        text: 'promotionCaptchaErrorText'
+      }
+    }
+
+    return {
+      title: 'promotionCaptchaTitle',
+      text: 'promotionCaptchaText'
+    }
+  }
+
+  render () {
+    const text = this.getText()
+
+    return <div
+      className={cx({
+        [css(styles.enabledContent__overlay, styles.enabledContent__captcha)]: true,
+        'enabledContent__overlay': true
```

Comment:
I meant:
is `styles.enabledContent__overlay` and the `enabledContent__overlay` below it reference the same styling? If so, then just:
`className={css(styles.enabledContent__overlay, styles.enabledContent__captcha)}`

Reply:
fixed